CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,3 @@
 - [cygnus-ngsi] Upgrade Debian version from 12.5 to 12.6 in Dockerfile
+- [cygnus-common][cygnus-ngsi] New setting mongo_uri (#2402)
```

Comment:
```suggestion
- [cygnus-common][cygnus-ngsi] New setting mongo_uri (#2402, #2387)
```

@AlvaroVega is ok that suggestion?

@kumardeepak5 please wait to @AlvaroVega confirmation before applying the suggestion.

Reply:
#2402 is a PR not a issue
Related issue with this PR is #2387 

CNR entry should be: [cygnus-common][cygnus-ngsi] New setting mongo_uri (#2387)

Reply:
Fixed in [85e58fb](https://github.com/telefonicaid/fiware-cygnus/pull/2402/commits/85e58fb8eaafe564f4241242748f74d29b9267b3)